### PR TITLE
Merge some more PRs

### PR DESCRIPTION
**Pull request by paultyng**
Includes #136, #141, and dependency updates.

### CODE REVIEW

**Comment by i-wilson** [comment]
Any updates on merging this one? I'd love to use outlier NRQL conditions

File: newrelic/resource_newrelic_nrql_alert_condition.go [hunk 1]
```diff
@@ -75,7 +83,7 @@ func resourceNewRelicNrqlAlertCondition() *schema.Resource {
 						"duration": {
 							Type:         schema.TypeInt,
 							Required:     true,
-							ValidateFunc: intInSlice([]int{1, 2, 3, 4, 5, 10, 15, 30, 60, 120}),
+							ValidateFunc: validation.IntAtLeast(1),
```
Comment (jamesgoodhouse) [inline newrelic/resource_newrelic_nrql_alert_condition.go]: 1 is not a valid value. Must be 2-120.

Comment (jamesgoodhouse) [inline newrelic/resource_newrelic_nrql_alert_condition.go]: 1 is not a valid value. Must be 2-120.

Comment (morganchristiansson) [reply newrelic/resource_newrelic_nrql_alert_condition.go]: This is fixed in PR #136 which is open since may

Comment (sanderblue) [reply newrelic/resource_newrelic_nrql_alert_condition.go]: #136 has been closed, however #178 addressed this and has been merged.

**Comment by sanderblue** [comment]
Addressed in #141, #144, and #178 (all merged).

**Pull request closed without merging**